hedwiki/score/HED_score_0.0.2.mediawiki

```diff
@@ -219,62 +219,62 @@ This project was supported by the by the National Institute of Mental Health of
         ** Respiration-oxygen-saturation
             *** <nowiki># {takesValue, valueClass=numericClass}</nowiki>
         ** Respiration-feature
-            *** Apnoe-respiration-feature <nowiki>[Add duration and comments in free text]</nowiki>
+            *** Apnoe-respiration <nowiki>[Add duration and comments in free text]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** Hypopnea-respiration-feature <nowiki>[Add duration and comments in free text]</nowiki>
+            *** Hypopnea-respiration <nowiki>[Add duration and comments in free text]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** Apnea-hypopnea-index-respiration-feature <nowiki>[events/h. Used with Findings-attributes tags]</nowiki>
+            *** Apnea-hypopnea-index-respiration <nowiki>[events/h. Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
             *** Periodic-respiration
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** Tachypnea <nowiki>[cycles per min. Used with Findings-attributes tags]</nowiki>
+            *** Tachypnea-respiration <nowiki>[cycles per min. Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** Other-respiration-features
+            *** Other-respiration-feature
                 **** <nowiki># {takesValue}[free text]</nowiki>
     * ECG <nowiki>[electrocardiography]</nowiki>
         ** ECG-QT-period
             *** <nowiki># {takesValue}[free text]</nowiki>
         ** ECG-feature
-            *** ECG-Sinus-rhythm <nowiki>[Normal rhythm. Used with Findings-attributes tags]</nowiki>
```

Comment:
The children of `ECG` don't satisfy the "is-a" relationship. Could we change `ECG` to `ECG-property`?